tests/main/theme-install/task.yaml

```diff
@@ -0,0 +1,40 @@
+summary: Verify that the theme REST API endpoints function correctly
+
+prepare: |
+    # Always use the curl snap, since 14.04 does not support --unix-socket
+    snap install --devmode --edge test-snapd-curl
+    snap install --edge jq
+
+restore: |
+    rm -f response.txt
+
+execute: |
+    echo "Install the gtk-common-themes snap"
+    snap install gtk-common-themes
+
+    echo "Check for presence of a collection of themes"
+    test-snapd-curl.curl -sSf -o response.txt --unix-socket /run/snapd.socket 'http://localhost/v2/themes?gtk-theme=Yaru&gtk-theme=TraditionalHumanized&icon-theme=Yaru&icon-theme=Adwaita&sound-theme=Yaru&sound-theme=No-Such-Theme'
```

Comment:
I didn't know you can repeat query arguments this way. Is that a standard method?

Reply:
Yes.  It's why the `url.Values` type used for query parameters uses `map[string][]string` as its underlying type.